### ISSUE TITLE
Document the required variables for NiftyPET AcquisitionData cosntructor

### ISSUE THREAD

**Issue by AnderBiguri** [CCPPETMR/SIRF#612]
Add it at least to `acquisition_data_from_scanner_info.py`, and any other place where info about niftyPET is available.

**Comment by KrisThielemans** [comment]
I don't know where you want to put this. As I wrote in [the email on the devel list](https://www.jiscmail.ac.uk/cgi-bin/webadmin?A2=CCP-PETMR-DEVEL;882e55b5.2004), `AcquisitionData` doesn't (and shouldn't) know about NiftyPET.

**Comment by AnderBiguri** [comment]
@KrisThielemans  but the user should, right? This is just an issue to add some text, somewhere in the examples, to document which are the required parameters if GPU code wants to be used.

**Comment by KrisThielemans** [comment]
agreed, but the question is where it needs to be documented... could indeed say in `acquisition_data_from_scanner_info.py` (and `.m`) something like "By default, the Siemens mMR uses acquisition settings corresponding to the following constructor". I'm afraid I don't really like mentioning NiftyPET in there, as 1) it isn't merged yet 2) it is probably a limitation of @rijobro 's NiftyPET wrapper 3) it creates dependencies.

of course, if there's a NiftyPET demo, then we put it there as well

**Comment by AnderBiguri** [comment]
That makes sense @KrisThielemans . Maybe we can leave this open until @rijobro's PR is merged?

**Comment by KrisThielemans** [comment]
Could just do the current demo already

**Comment by AnderBiguri** [comment]
@KrisThielemans certainly, I meant keep it open even after current needed modifications are put (the demo) , until the PR is merged.
